property_test.go

```diff
@@ -107,6 +110,26 @@ func TestProperty(t *testing.T) {
 		require.False(t, ok)
 		require.Nil(t, property1)
 	})
+
+	t.Run("Info", func(t *testing.T) {
```

Comment:
The following cases should be covered:
- an error with a property and underlying, underlying goes last
- a chain of err2 -> err1 with property -> err0, properties of err1 go before the innermost error err0 in message text
- non-printable property is not in the message
- in chain like err1 -> err0 with property, properties are not inherited when formatting

For readability of tests as parts of documentation, it is better to test those cases (and some more basic ones, too) separately rather than all in one test.